packages/core/src/modules/indy/services/IndyHolderService.ts

```diff
@@ -201,7 +201,7 @@ export interface StoreCredentialOptions {
   credential: Indy.Cred
   credentialDefinition: Indy.CredDef
   credentialId?: Indy.CredentialId
-  revocationRegistryDefinitions?: Indy.RevRegsDefs
+  revocationRegistryDefinitions?: Indy.RevocRegDef
```

Comment:
```suggestion
  revocationRegistryDefinitions?: Indy.RevocRegDefs
```

Reply:
It should actually be `RevocRegDef` otherwise there is an error saying type `RevocRegDefs` of `revocationRegistryDefinitions ?? null `does not match `RevocRegDef`
`  /**
   * Store a credential in the wallet.
   *
   * @returns The credential id
   */
  public async storeCredential({
    credentialRequestMetadata,
    credential,
    credentialDefinition,
    credentialId,
    revocationRegistryDefinitions,
  }: StoreCredentialOptions): Promise<Indy.CredentialId> {
    try {
      return await this.indy.proverStoreCredential(
        this.wallet.handle,
        credentialId ?? null,
        credentialRequestMetadata,
        credential,
        credentialDefinition,
        revocationRegistryDefinitions ?? null
      )
    } catch (error) {
      throw new IndySdkError(error)
    }
  }`